packages/vitest/README.md

```diff
@@ -0,0 +1,84 @@
+# ![@knapsack-pro/vitest](./img/logo.png)
+
+<h3 align="center">Speed up your tests</h3>
+<p align="center">Run your 1-hour test suite in 2 minutes with optimal parallelisation on your existing CI infrastructure</p>
+
+---
+
+<div align="center">
+  <a href="https://circleci.com/gh/KnapsackPro/knapsack-pro-js">
+    <img alt="Circle CI" src="https://circleci.com/gh/KnapsackPro/knapsack-pro-js.svg?style=svg" />
+  </a>
+</div>
+
+<br />
+<br />
+
+Knapsack Pro wraps [Vitest](https://vitest.dev/) and works with your existing CI infrastructure to parallelize tests optimally:
+
+- Dynamically splits your tests based on up-to-date test execution data
+- Is designed from the ground up for CI and supports all of them
+- Tracks your CI builds to detect bottlenecks
+- Does not have access to your source code and collects minimal test data
+- Enables you to export historical metrics about your CI builds
+- Replaces local dependencies like Redis with an API and runs your tests regardless of network problems
+
+## Getting Started
+
+1. Install the package
+
+```
+npm install @knapsack-pro/vitest
+```
+
+2. Update your CI to use a matrix build. See the [example workflow](.github/workflows/vitest-example-test-suite.yaml) for reference.
+3. Replace `vitest` with `@knapsack-pro/vitest` in your CI config
+4. Set the `KNAPSACK_PRO_TEST_SUITE_TOKEN` environment variable to your Knapsack Pro test suite token, and optionally set other environemnt variables as needed.
+
+## Environment variables
+Below are the primary environment variables you may need to set to use Knapsack Pro with Vitest.  For a full list of environment variables, see [Knapsack Pro Jest](https://docs.knapsackpro.com/jest/reference/#environment-variables).
```

Comment:
I made a mistake. The `#environment-variables` anchor does not exit. Let's do this:

```suggestion
Below are the primary environment variables you may need to set to use Knapsack Pro with Vitest.  For a full list of environment variables, see [Knapsack Pro Jest](https://docs.knapsackpro.com/jest/reference/).
```